Tests/Fixtures/TranslatableItemI18n.php

```diff
@@ -33,6 +33,11 @@ public function __construct($id = null, $locale = null, $value = null)
         $this->value = $value;
     }
 
+    public function isPrimaryKeyNull(): bool
+    {
+        return false;
```

Comment:
🔧 Same here: Please add the proper check.
```suggestion
        return null === $this->getId();
```